documentation/RELEASE_NOTES.md

```diff
@@ -38,6 +38,145 @@ For example this will remove the record named "unwanted":
 record("#", "unwanted") { }
 ```
 
+### Access Security: Certificate-based Authorization
+
+Added a new option and two new fields to `ASG(){RULE(...,<new_option>) {<new_fields>}}`, and added a new access type, `RPC`, to ASG(){RULE(<new_type>)}.  Three new API functions were also added to the `libcom` library to support these new features.  The ability to provide ACF files in EPICS YAML format has been added.
+
+There are no external dependencies for these features.
+
+#### New Option
+- `ISTLS` - Controls PV access authorization based on whether the connection is using TLS.  If present then the connection must be using TLS.  If absent the connection type is ignored unless specifically set to false using a yaml configuration.
+
+```
+ASG(ENCRYPTED) {
+    RULE(1, WRITE, ISTLS)
+}
+```
+
+In yaml you have more control over the `isTLS` option.  You can mandate that the connection be TLS or that it not be TLS:
+
+```yaml
+# EPICS YAML
+version: 1.0
+asgs:
+  # Write access with a secure connection
+  - name: ENCRYPTED
+    rules:
+      - level: 1
+        access: WRITE
+        isTLS: true
+```
+
+```yaml
+# EPICS YAML
+version: 1.0
+asgs:
+  # Read access with a non-secure connection
+  - name: ENCRYPTED
+    rules:
+      - level: 1
+        access: READ
+        isTLS: false
+```
+
+#### New Fields
+- `METHOD("type"[,...])` - Controls PV access authorization based on the connection method (e.g. "x509", "ca", "anonymous").  True if any match.  "x509" is for connections that have been established by and identity authenticated with an X509 certificate.
+- `AUTHORITY("common_name"[,...])` - Controls PV access authorization based on the Common Name of the Certificate Authority that signed the client's certificate.  True if any match.  Only applies to connections that have been established by an identity authenticated with an X509 certificate.
+
+These fields accept comma-separated lists and work with Secure PVAccess connections. Multiple entries in each field act as a logical OR. For example to mandate Stanford National Laboratory or Stanford University issued certificate-based authentication use the following rule:
+
+```
+ASG(secure) {
+    RULE(1, WRITE) {
+        METHOD("x509")
+        AUTHORITY("SLAC.STANFORD.EDU CA", "STANFORD.EDU CA")
+    }
+}
```

Comment:
Core Group:

```
UAG(people) { ... }
ASG(secure) {
    RULE(1, WRITE) {
        UAG(people)
        METHOD("x509")
        AUTHORITY("SLAC.STANFORD.EDU CA", "STANFORD.EDU CA")
    }
}
```

Should `METHOD` and `AUTHORITY` be part of `UAG()`?

What about `HAG`?